docs/integrations/nestjs.md

```diff
@@ -0,0 +1,287 @@
+---
+sidebar_position: 2
+title: Nest.js
+---
+import InstallNode from '/docs/partials/_install_node.mdx';
+
+This guide shows you how to integrate [DBOS Transact](https://github.com/dbos-inc/dbos-transact) with your existing [Nest.js](https://nestjs.com/) application.
+
+Nest.js uses Dependency Injection (an [inversion of control](https://en.wikipedia.org/wiki/Inversion_of_control) technique) and encourages you to write your business logic in services that can be injected into your controllers or other services. Such injectable services are exported by Nest.js modules as "providers".
```

Comment:
Don't need this paragraph or the next one, Nest.js users know what Nest.js is.

Reply:
Instead, add another sentence to the first paragraph that briefly says why you want to add DBOS to your Nest.js app.